pkg/token/options.go

```diff
@@ -34,6 +34,7 @@ const (
 	defaultEnvironmentName = "AzurePublicCloud"
 
 	DeviceCodeLogin       = "devicecode"
+	InteractiveLogin      = "interactive"
```

Comment:
i'm actualy torn between this or `interactivebrowser`. @bcho WDYT?

Reply:
I think we better to follow other SDK / docs to use `interactive`. Example: https://learn.microsoft.com/en-us/python/api/msal/msal.application.publicclientapplication?view=azure-python#msal-application-publicclientapplication-acquire-token-interactive